core/src/main/scala/quasar/fs/Natural.scala

```diff
@@ -21,12 +21,19 @@ final class Natural private (val value: Long) {
 
   def toInt: Int =
     value.toInt
+
+  override def equals(other: scala.Any) = other match {
```

Comment:
We should probably keep the "normal" equals though right? Or else it won't work with specs2, or am I missing something?